readme.md

```diff
@@ -38,7 +50,7 @@ Describe the differences between a SQL and NoSQL DB, and when you might use each
 
 Your Answer:
 ```text
-
+ a noSQL db is usually used when there is only one to many relationships or very few many to many relationships (if any at all).  SQL databases have more structural options and can handle the many different types of relationships including many to many.  In a noSQL db a program can handle the calls to the db for you where as with a SQL db you would have to set up the sql calls.
```

Comment:
:+1:

The last line is unclear for me.  Mongoose and ActiveRecord are both programs that handle the calls to the db.